.githooks/pre-push

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+set -e
+
+pnpm lint
+pnpm check
```

Comment:
Can we have a trailing newline here please? This is going to bug me.

Reply:
done